### ISSUE TITLE
[Help] Cant Set Selected Value For Multiple Select with Other Initialization Paramaters

### ISSUE THREAD

**Issue by cupcakedream** [fomantic/Fomantic-UI#1470]
Hey All - 

I'm trying to set selected values for a multiple select dropdown but I can't seem to figure out how to do this during initialization. I know I can do it after initialize but I want to avoid firing an onchange event at all costs. I've looked everywhere can't seem to figure out how to do this. Is it not supported?

```
    jQuery('.outcome select').dropdown({
        selected : [1,2,3,4,5],
        onChange : function( x,y,z ) {
            jQuery.ajax({ url : 'ajax.php', method : 'post',
                data : { action : 'outcome', id : y }
            });
        }
    });
```

**Comment by lubber-de** [comment]
I think you found a bug. 
You can already provide preselected values during initialization:
1. By converting a `select` tag and having the desired `option` values of that original select having the `selected property`

```html
<select class="ui dropdown" multiple="">
  <option value="">State</option>
  <option selected value="1">one</option>
  <option value="2">two</option>
  <option selected value="3">three</option>
</select>
```
```javascript
```

2. By providing the `selected` attribute to the desired fields of the given `values` setting
```javascript
$('.ui.dropdown').dropdown({
  values:[{
    value:1,
    name: 'one',
    selected: true
  },{
    value:2,
    name: 'two'
  },{
    value:3,
    name: 'three',
    selected: true
  }],
});
```

Both options will define the preselected values without the need of an additional `set selected` behavior call after initialization.
However, (and this is why i will tag this as a bug) using the `values` setting will call the `onChange` event for each selected value, which should be fetched because we already have a `fireOnInit` setting, which should prevent this

When using option 1 (converting a select) you are ready to go: The previous selected values are preselected without triggering the `onChange` callback

For option 2 i will prepare a PR to fix this

See
https://jsfiddle.net/7g2ywtfc/

**Comment by lubber-de** [comment]
Fixed it by #1471 
See https://jsfiddle.net/7g2ywtfc/1/

**Comment by cupcakedream** [comment]
Awesome thanks!
